src/index.ts

```diff
@@ -50,6 +50,5 @@ export function start(options: DebugAgentConfig|StackdriverConfig): Debuglet|
   debuglet = new Debuglet(debug, agentConfig);
   debuglet.start();
 
-  // We return the debuglet to facilitate testing.
-  return agentConfig.testMode_ ? debuglet : undefined;
+  return debuglet;
```

Comment:
Why did this line change?

Reply:
because the user will call 
```javascript
const debug = require('@google/cloud-debugger').start();
const debugReady = debug.isReady();
```
we cannot return undefined in `start`

Reply:
The type definition for `start` should be updated. Secondly, let's return an opaque object with an `isReady` function rather than exposing the full `debuglet`.

Reply:
Done. Added an IsReadyManager class to only expose isReady.